tests/ui/plugins/testFooterPlugin.spec.ts

```diff
@@ -72,63 +71,73 @@ test.describe('Footer plugin tests', () => {
       await expect(page.getByTestId(testIDs.plugins.footer.main)).not.toBeVisible()
     })
   })
-  sites.forEach((site: string) => {
-    test(`Footer plugin must be visible on site ${site} when active`, async ({ page }) => {
-      // GIVEN: The footer plugin is active
-      await page.route('*/**/api/plugins/footer/', (route) =>
-        route.fulfill({
-          json: footerEnabledDataMock,
-        })
-      )
-      // WHEN: The user navigates to the tested page
-      await page.goto(site)
-      await page.waitForLoadState()
+  ;[true, false].forEach((showCopyright: boolean) => {
+    sites.forEach((site: string) => {
+      test(`Footer plugin must be visible on site ${site} when active with copyright ${showCopyright ? 'enabled' : 'disabled'}`, async ({
```

Comment:
```suggestion
      test(`Footer plugin must ${showCopyright ? '' : 'not'} be visible on site ${site} when copyright is ${showCopyright ? 'enabled' : 'disabled'}`, async ({
```
So ists mMn eindeutiger als Beschreibung, nicht?